app/controllers/AppealsController.scala

```diff
@@ -178,13 +178,13 @@ class AppealsController @Inject()(val appConfig: AppConfig,
                     Ok(responseModel.caseID)
                   case status =>
                     PagerDutyHelper.logStatusCode("submitAppeal", status)(RECEIVED_4XX_FROM_FILE_NOTIFICATION_ORCHESTRATOR, RECEIVED_5XX_FROM_FILE_NOTIFICATION_ORCHESTRATOR)
-                    logger.error(s"[AppealsController][submitAppeal] - Received unknown response ($status) from file notification orchestrator. Response body: ${response.body}")
+                    logger.error(s"[AppealsController][submitAppeal] Unable to store file notification for user with enrollment: $enrolmentKey penalty $penaltyNumber - Received unknown response ($status) from file notification orchestrator. Response body: ${response.body}")
                     auditStorageFailureOfFileNotifications(seqOfNotifications)
                     returnErrorResponseIfMultiAppeal(isMultiAppeal)(s"Appeal submitted (case ID: ${responseModel.caseID}) but received $status response from file notification orchestrator")(responseModel.caseID)
                 }
             }.recover {
               case e => {
-                logger.error(s"[AppealsController][submitAppeal] - An unknown exception occurred when attempting to store file notifications, with error: ${e.getMessage}")
+                logger.error(s"[AppealsController][submitAppeal] Unable to store file notification for user with enrollment: $enrolmentKey penalty $penaltyNumber - An unknown exception occurred when attempting to store file notifications, with error: ${e.getMessage}")
```

Comment:
This change needs to be reflected in the unit tests, on line 1110

Reply:
Fixed ✅